frontend/src/metabase/routes-public.jsx

```diff
@@ -16,7 +16,7 @@ export const getRoutes = store => {
         <Route path="question/:uuid" component={PublicOrEmbeddedQuestion} />
         <Route
           path="dashboard/:uuid(/:tabSlug)"
-          component={PublicOrEmbeddedDashboardControlled}
+          component={PublicOrEmbeddedDashboardPage}
```

Comment:
I think there are `PublicOrEmbeddedDashboardPage`, `PublicOrEmbeddedDashboardControlled`, `PublicOrEmbeddedDashboardView` and `PublicOrEmbeddedDashboard` at this moment. Should we document their relationships, as I think it's getting a bit hierarchical?

Reply:
Yeah good point, I'll do that after I finish with the refactor so that I can write some coherent comments. We should also be able to removed the *Controlled ones by then